prov/shm/src/smr_progress.c

```diff
@@ -569,29 +569,47 @@ static int smr_progress_ipc(struct smr_cmd *cmd, enum fi_hmem_iface iface,
 }
 
 static void smr_do_atomic(void *src, void *dst, void *cmp, enum fi_datatype datatype,
-			  enum fi_op op, size_t cnt, uint16_t flags)
+			  enum fi_op op, size_t cnt, uint16_t flags, enum fi_hmem_iface iface, uint64_t device)
```

Comment:
Please keep line lengths to about 80 characters for better readability. Comment applies throughout patch.

Reply:
done